README.rst

```diff
@@ -1,32 +1,28 @@
-+----------------------+----------------------------------------------------------------+
-| master branch status | |BuildStatus| |Docs| |MITLicense| |CoverageStatus|             |
-+----------------------+----------------------------------------------------------------+
-| GitHub status        | |OpenIssues| |OpenPullRequests|                                |
-+----------------------+------------+---------------------------------------------------+
-| Versions             | NI-DCPower | |nidcpowerLatestVersion| |nidcpowerPythonVersion| |
-|                      +------------+---------------------------------------------------+
-|                      | NI-DMM     | |nidmmLatestVersion| |nidmmPythonVersion|         |
-|                      +------------+---------------------------------------------------+
-|                      | NI-FGEN    | |nifgenLatestVersion| |nifgenPythonVersion|       |
-|                      +------------+---------------------------------------------------+
-|                      | NI-SCOPE   | |niscopeLatestVersion| |niscopePythonVersion|     |
-|                      +------------+---------------------------------------------------+
-|                      | NI-SWITCH  | |niswitchLatestVersion| |niswitchPythonVersion|   |
-|                      +------------+---------------------------------------------------+
-|                      | NI-ModInst | |nimodinstLatestVersion| |nimodinstPythonVersion| |
-+----------------------+------------+---------------------------------------------------+
++----------------------+---------------------------------------------------------------------------------------------+
+| master branch status | |BuildStatus| |Docs| |MITLicense| |CoverageStatus|                                          |
++----------------------+---------------------------------------------------------------------------------------------+
+| GitHub status        | |OpenIssues| |OpenPullRequests|                                                             |
++----------------------+------------+--------------------+----------------------------+------------------------------+
+|                      | **Name**   | **Driver Version** | **PyPI Version**           | **Supported Python Version** |
```

Comment:
To a customer, **Driver Version** conveys that _this_ version is the one that works with the bindings. That's misleading. What is it really?

1. The version we extracted metadata from
2. The version we tested against

I think it's the second one, right? What are we really trying to tell customers here?